server/tinylicious/package.json

```diff
@@ -39,18 +39,18 @@
 	},
 	"dependencies": {
 		"@fluidframework/common-utils": "^1.1.1",
-		"@fluidframework/gitresources": "^0.1039.1000",
-		"@fluidframework/protocol-base": "^0.1039.1000",
+		"@fluidframework/gitresources": "^0.1040.1000",
```

Comment:
this is a breaking change to tinylicious and requires a major version bump

Reply:
Yes, I'll do a major bump in tinylicious from main and revert the bump in the server dependencies here.